lib/index.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
     CronService: require('./cron').CronService,
     HttpService: require('./http').HttpService,
     SlackService: require('./slack').SlackService,
+    SQLStoreStategy: require('./sqlStoreStategy').SQLStoreStategy
```

Comment:
The name strategy is confusing. Maybe "backend"?

Reply:
what does backend mean? everything here is backend @avnerbraverman. I took Strategy from npm Passport.